django/db/backends/mysql/features.py

```diff
@@ -64,23 +64,11 @@ def minimum_database_version(self):
         else:
             return (8,)
 
-    @cached_property
-    def test_collations(self):
-        charset = "utf8"
-        if (
-            self.connection.mysql_is_mariadb
-            and self.connection.mysql_version >= (10, 6)
-        ) or (
-            not self.connection.mysql_is_mariadb
-            and self.connection.mysql_version >= (8, 0, 30)
-        ):
-            # utf8 is an alias for utf8mb3 in MariaDB 10.6+ and MySQL 8.0.30+.
-            charset = "utf8mb3"
-        return {
-            "ci": f"{charset}_general_ci",
-            "non_default": f"{charset}_esperanto_ci",
-            "swedish_ci": f"{charset}_swedish_ci",
-        }
+    test_collations = {
+        "ci": "utf8mb3_general_ci",
+        "non_default": "utf8mb3_esperanto_ci",
+        "swedish_ci": "utf8mb3_swedish_ci",
+    }
```

Comment:
Alternatively, we could just ignore the `RuntimeWarning` in `tests/inspectdb/models.py` like in https://github.com/django/django/blob/e20306a213c775cfb1096c07eef976c548567ba1/tests/inspectdb/models.py#L99-L105

Reply:
Never mind. I was hoping that it would just work with MariaDB if it works with MySQL ☹️